tools/roslaunch/test/xml/test-arg-valid-include.xml

```diff
@@ -2,5 +2,8 @@
   <arg name="another_parameter_not_used" value="dummy"/>
   <arg name="grounded" value="not_set"/>
   <include file="$(find roslaunch)/test/xml/test-arg-invalid-included.xml" pass_all_args="true"/>
+
+  <param name="file" value="test-arg-invalid-included.xml"/>
+  <include file="$(find roslaunch)/test/xml/$(param file)" pass_all_args="true"/>
```

Comment:
nitpick: this test seems out of place. This file is used to test setting args in included launch files. Also, including the same file twice here seems odd.

What are you trying to test? Perhaps it should be moved to a dedicated test.

Reply:
Probably I wanted to illustrate the use of param substitution in an include statement.

Reply:
I think we should create a separate launch file to test for this case, instead of adding onto this existing one.